projects/graph/src/graph.py

```diff
@@ -29,6 +29,23 @@ def bft(self, vertex):
                         continue
                     queue.append(vertex)
                 visited.append(queue.pop(0))
-            return(visited)
+            print(visited)
+        else:
+            print("Vertex not in list")
+
+    def dft(self, vertex):
+        queue = []
+        visited = []
```

Comment:
Hoping you were able to take into the fundamental implementation I was speaking, if you still have trouble processing this please let me know